docs/_pages/about.md

```diff
@@ -4,11 +4,11 @@ permalink: /about/
 header:
   overlay_color: "#000"
   overlay_filter: "0.5"
-  overlay_image: /assets/images/conversational-ai.jpg
+  overlay_image: /assets/images/amc_banner.jpg
```

Comment:
You need to rename `amc_banner.jpg` to `amc-banner.jpg` to prevent 404 error.

Reply:
I failed CI because of the image file name `AMC-banner`.  I have corrected it in my new commit.